src/server/rest/v1/service/UserService.ts

```diff
@@ -388,30 +410,32 @@ export default class UserService {
     next();
   }
 
-  public static async handleGetUsersInError(action: ServerAction, req: Request, res: Response, next: NextFunction): Promise<void> {
+  public static async handleGetUsersInError(action: ServerAction, req: Request, res: Response, next: NextFunction): Promise<DataResult<User>> {
     // Filter
     const filteredRequest = UserValidatorRest.getInstance().validateUsersInErrorGetReq(req.query);
     // Get authorization filters
-    const authorizations = await AuthorizationService.checkAndGetUsersInErrorAuthorizations(
-      req.tenant, req.user, filteredRequest);
+    const authorizationUserInErrorFilters = await AuthorizationService.checkAndGetUsersInErrorAuthorizations(req.tenant, req.user, filteredRequest);
+    if (!authorizationUserInErrorFilters.authorized) {
+      return Constants.DB_EMPTY_DATA_RESULT;
+    }
     // Get users
     const users = await UserStorage.getUsersInError(req.tenant,
       {
         search: filteredRequest.Search,
         roles: (filteredRequest.Role ? filteredRequest.Role.split('|') : null),
         errorTypes: (filteredRequest.ErrorType ? filteredRequest.ErrorType.split('|') : Object.values(UserInErrorType)),
-        ...authorizations.filters
+        ...authorizationUserInErrorFilters.filters
       },
       {
         limit: filteredRequest.Limit,
         onlyRecordCount: filteredRequest.OnlyRecordCount,
         skip: filteredRequest.Skip,
         sort: UtilsService.httpSortFieldsToMongoDB(filteredRequest.SortFields)
       },
-      authorizations.projectFields
+      authorizationUserInErrorFilters.projectFields
     );
     // Add Auth flags
-    await AuthorizationService.addUsersAuthorizations(req.tenant, req.user, users as UserDataResult, authorizations);
+    await AuthorizationService.addUsersAuthorizations(req.tenant, req.user, users as UserDataResult, authorizationUserInErrorFilters);
```

Comment:
In handleImportUsers() use new auth, limit the use of any Authorizations.can() in the service

Reply:
forgot this one 😑 done ✅